client/src/components/ProfileSidebar/Sidebar.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import Stack from '@mui/material/Stack';
+
+import useStyles from './useStyles';
+function Sidebar(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Stack className={classes.main} alignItems="center">
+      <Button component={Link} to="/edit-profile">
+        Edit Profile
+      </Button>
+
+      <Button component={Link} to="/profile">
```

Comment:
Also, look into the usage of NavLink so that we can apply CSS classes when the link is active ;)

Reply:
Sure, Thanks